src/__myproject__/_orchestration/clickhouse_export.py

```diff
@@ -0,0 +1,110 @@
+# Databricks notebook source
+# MAGIC %run ../app/bootstrap
+
+# COMMAND ----------
+
+from pyspark.sql import utils, SparkSession, DataFrame
+from pyspark.sql import functions as f
+from pyspark.dbutils import DBUtils
+from clickhouse_driver import Client as ClickhouseClient
+from logging import Logger
+import daipe as dp
+
+# COMMAND ----------
+
+entity = dp.fs.get_entity()
+
+# COMMAND ----------
+
+@dp.notebook_function()
+def init_widgets(widgets: dp.Widgets):
+    widgets.add_text("config_url", "", "Config url")
+
+# COMMAND ----------
+
+@dp.notebook_function(dp.get_widget_value("config_url"))
+def load_config(config_url: str, logger: Logger, dbutils: DBUtils, spark: SparkSession):
+    try:
+        config = spark.read.json(config_url).collect()[0].asDict()
+    except utils.IllegalArgumentException:
+        logger.error("Config path not set.")
+        dbutils.notebook.exit(0)
+        return None
+    except utils.AnalysisException:
+        logger.error(f"Couldn't find config at {config_url}.")
+        dbutils.notebook.exit(0)
+        return None
+    else:
+        logger.info(f"Loaded config from {config_url}")
+        return config
+
+# COMMAND ----------
+
+current_env = load_config.result['env']
```

Comment:
these global variables really shouldn't be here, maybe make another @notebook_function and return a dict?